spikeinterface/sorters/runsorter.py

```diff
@@ -144,22 +150,30 @@ class ContainerClient:
       * docker with "docker" python package
       * singularity with  "spython" python package
     """
-    def __init__(self, mode, container_image, volumes, extra_kwargs):
+    def __init__(self, mode, container_image, volumes, gpu_to_use=None):
         assert mode in ('docker', 'singularity')
         self.mode = mode
 
         if mode == 'docker':
             import docker
             client = docker.from_env()
-            if extra_kwargs.get('requires_gpu', False):
-                extra_kwargs.pop('requires_gpu')
-                extra_kwargs["device_requests"] = [
-                    docker.types.DeviceRequest(count=-1, capabilities=[['gpu']])]
+
+            if gpu_to_use:
+                gpu_to_use = [gpu_to_use] if isinstance(gpu_to_use, int) else gpu_to_use
+                device_requests = [
+                    docker.types.DeviceRequest(
+                        device_ids=, count=len(gpu_to_use), capabilities=[['gpu']]
```

Comment:
Should this be as follows?:
```suggestion
                        device_ids=gpu_to_use, count=len(gpu_to_use), capabilities=[['gpu']]
```